docs/src/main/asciidoc/_building.adoc

```diff
@@ -1,5 +1,26 @@
 == How to Build Spring Cloud Contract
 
+=== Cloning the repository on Windows
+
+While cloning this project on *windows*, some files in the git repository may exceed the Windows maximum file path limit of 255 characters, which may
+result in improper/half checked out repository.
+
+* To resolve this the `*core.longPaths=true*` git option should be set:
+
+```
+git config --system core.longpaths true
+```
+
+*_OR_*
```

Comment:
Change to:

The following example shows how to clone Spring Cloud Contract:

Reply:
Since I am having an awful time seeing how the lines relate to one another, here's what the whole thing should be (note that there are no bullets anywhere):

------------------

While cloning this project on Windows, some files in the git repository may exceed the Windows maximum file path limit of 255 characters, which may result in an incorrectly (probably partially) checked out repository.

To resolve this issue, you can set the `core.longPaths attribute` to `true` or clone the Spring Cloud Contract repository.

The following example shows how to set the core.longPaths attribute to true:

```git config --system core.longpaths true```

The following example shows how to clone Spring Cloud Contract:

```git clone -c core.longPaths=true https://github.com/spring-cloud/spring-cloud-contract.git```

Reply:
@Buzzardo It's now corrected.